exercises/say/makefile

```diff
@@ -0,0 +1,25 @@
+CFLAGS  = -std=c99
+CFLAGS += -g
+CFLAGS += -Wall
+CFLAGS += -Wextra
+CFLAGS += -pedantic
+CFLAGS += -Werror
+
+VFLAGS  = --quiet
+VFLAGS += --tool=memcheck
+VFLAGS += --leak-check=full
+VFLAGS += --error-exitcode=1
+
+test: tests.out
+	@./tests.out
+
+memcheck: tests.out
+	@valgrind $(VFLAGS) ./tests.out
+	@echo "Memory check passed"
+
+clean:
+	rm -rf *.o *.out *.out.dSYM
+
+tests.out: test/test_say.c src/say.c src/say.h
+	@echo Compiling $@
+	@cc $(CFLAGS) src/say.c test/vendor/unity.c test/test_say.c -o tests.out -lm
```

Comment:
`-lm` should probably be added by the variable `LDFLAGS` (doesn't currently exist). See [https://stackoverflow.com/questions/13249610/how-to-use-ldflags-in-makefile](https://stackoverflow.com/questions/13249610/how-to-use-ldflags-in-makefile) for an example of `LDFLAGS`